huaweicloud/resource_huaweicloud_apig_instance.go

```diff
@@ -154,8 +154,14 @@ func buildMaintainEndTime(maintainStart string) (string, error) {
 	return fmt.Sprintf("%02d:00:00", (num+4)%24), nil
 }
 
+// Method buildRFC3339Timestamp is used to unify the time format to RFC-3339 and return a time string.
+func buildRFC3339Timestamp(timestamp int64) string {
+	createTime := time.Unix(timestamp, 0)
+	return createTime.Format(time.RFC3339)
+}
```

Comment:
move to utils directory ? @Jason-Zhang9309 What do you think about this ?

Reply:
move to utils.go will be more clearly.
also we can get a better name like FormatTimeStamp